src/providers/extraction_providers/visionExtractionProvider.ts

```diff
@@ -0,0 +1,105 @@
+import { ExtractionProvider } from './extractionProvider';
+import { Image, ExtractedImage } from '../../models/data_models/image';
+import { AssignedLabel, DetectedObject, Coordinate } from '../../models/data_models/extraction';
+
+import vision = require('@google-cloud/vision');
```

Comment:
Can't you do this:
```suggestion
import vision from '@google-cloud/vision';
```

Reply:
I chose my version to allow for more easily specifying types, for example I can now say 
```label: vision.protos.google.cloud.vision.v1.IEntityAnnotation```
whereas otherwise I would get a namespace not found error.

Reply:
This line of code was taken from Google's quick start linked [here](https://cloud.google.com/vision/docs/quickstart-client-libraries#using_the_client_library). I am guessing that the Vision API was implemented and exported with an `export = ` statement, which according to the Typescript docs linked [here](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require), require a `import module = require("module")` to import